underfs/s3/src/test/java/alluxio/underfs/s3/S3UnderFileSystemTest.java

```diff
@@ -14,22 +14,27 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.powermock.api.mockito.PowerMockito;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 import org.powermock.reflect.Whitebox;
 
 /**
  * Tests for the private helper methods in {@link S3UnderFileSystem} that do not require an S3
  * backend.
  */
-public class S3UnderFileSystemTest {
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(S3UnderFileSystem.class)
+public final class S3UnderFileSystemTest {
```

Comment:
would it be difficult to add a test that checks that the chunked listing works (e.g. by setting LISTING_LENGTH to 1 and trying to list a couple on files)